bitmap-ram-rod/lib/transformer.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+const transform = module.exports = {};
+
+transform.modify = (parent, blueCallback, greenCallback, redCallback) => {
+
+  const helper = (bmp, position) => (callback) => (...ind) => {
```

Comment:
This is a very interesting construct on how to do this so props for creativity. You may want to be careful that you aren't adding in complexity just to be complex and show off what you can do. There are definitely simpler ways to do this that will work just as well and won't add as many calls to the stack and layer this so much.